src/shallow.js

```diff
@@ -11,13 +12,31 @@ function nodeToJson(node) {
         return node;
     }
 
+    if (!isElement(node)) {
+        if (Array.isArray(node)) {
+            return node.map(nodeToJson);
+        }
+
+        if (typeof node === 'object') {
+            const obj = {};
+
+            Object.keys(node).forEach(key => {
+                obj[key] = nodeToJson(node[key]);
+            });
+
+            return obj;
```

Comment:
Ah sorry, tests are failing because `Object.entries` is not available in Node v4. Can you use [object.entries](https://github.com/es-shims/Object.entries)?

```js
import entries from 'object.entries';

// ...

return entries(node).reduce(...
```